internal/services/compute/orchestrated_virtual_machine_scale_set.go

```diff
@@ -496,22 +497,65 @@ func OrchestratedVirtualMachineScaleSetDataDiskSchema() *pluginsdk.Schema {
 					Default:  false,
 				},
 
-				// TODO 3.0 - change this to ultra_ssd_disk_iops_read_write
+				//TODO: Remove the attribute below in 4.0
 				"disk_iops_read_write": {
 					Type:     pluginsdk.TypeInt,
 					Optional: true,
 					Computed: true,
+					ConflictsWith: func() []string {
+						if features.ThreePointOhBeta() {
+							return []string{"ultra_ssd_disk_iops_read_write"}
+						}
+						return []string{}
+					}(),
+					Deprecated: func() string {
+						if features.ThreePointOhBeta() {
+							return "This property has been renamed to `ultra_ssd_disk_iops_read_write` and will be removed in v3.0 of the provider"
+						}
+						return ""
+					}(),
 				},
 
-				// TODO 3.0 - change this to ultra_ssd_disk_iops_read_write
+				//TODO: Remove the attribute below in 4.0
 				"disk_mbps_read_write": {
 					Type:     pluginsdk.TypeInt,
 					Optional: true,
 					Computed: true,
+					ConflictsWith: func() []string {
+						if features.ThreePointOhBeta() {
+							return []string{"ultra_ssd_mbps_read_write"}
+						}
+						return []string{}
+					}(),
+					Deprecated: func() string {
+						if features.ThreePointOhBeta() {
+							return "This property has been renamed to `ultra_ssd_disk_mbps_read_write` and will be removed in v3.0 of the provider"
+						}
+						return ""
+					}(),
 				},
 			},
 		},
 	}
+
+	o := out.Elem.(*pluginsdk.Resource).Schema
+	if features.ThreePointOhBeta() {
+		o["ultra_ssd_disk_iops_read_write"] = &pluginsdk.Schema{
+			Type:          pluginsdk.TypeInt,
+			Optional:      true,
+			Computed:      true,
+			ConflictsWith: []string{"disk_iops_read_write"},
+		}
+
+		o["ultra_ssd_mbps_read_write"] = &pluginsdk.Schema{
+			Type:          pluginsdk.TypeInt,
+			Optional:      true,
+			Computed:      true,
+			ConflictsWith: []string{"disk_mbps_read_write"},
+		}
+	}
```

Comment:
as whit the other pr, is there a reason to not just deprecate this now and add the new properties and then in a couple weeks just remove the in 3.0? these are not going to be highly used properties